backend/apps/accounts/db_stats.py

```diff
@@ -0,0 +1,97 @@
+from apps.ineedstudent.models import Hospital
+from apps.iamstudent.models import Student, EmailGroup, EmailToSend, EmailToHospital
+from .models import User, Newsletter
+
+from apps.iamstudent.models import AUSBILDUNGS_TYPEN_COLUMNS
+from django.db.models import Count
+from apps.iamstudent.forms import form_labels
+
+from django.utils.translation import gettext_lazy as _
+
+
+class RegisterList(list):
+    def register(self, method):
+        self.append(method)
+        return method
+
+
+class DataBaseStats:
+
+    stat_count = RegisterList()
+    stat_list = RegisterList()
+
+    # todo: last X days?
+
+    @stat_count.register
+    def admin_count(self):
+        return (_("Aktive Staffmember"), User.objects.filter(is_staff=True).count())
+
+    @stat_count.register
+    def approved_hospital_count(self):
+        return (
+            _("Akzeptierte Institutionen"),
+            Hospital.objects.filter(is_approved=True, user__validated_email=True).count(),
+        )
+
+    @stat_count.register
+    def validated_student_count(self):
+        return (
+            _("Registrierte Helfende"),
+            Student.objects.filter(user__validated_email=True).count(),
+        )
+
+    @stat_count.register
+    def deactivated_accounts(self):
+        return (
+            _("Anzahl deaktivierter Helfenden"),
+            Student.objects.filter(is_activated=True).count(),
```

Comment:
Shouldn't that be `False`?
```suggestion
            Student.objects.filter(is_activated=False).count(),
```